.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,39 @@
+
+name: "Static Analysis"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+  push:
+    branches:
+      - "*.x"
+
+jobs:
+  static-analysis-psalm:
+    name: "Static Analysis with Psalm"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
```

Comment:
I haven't given much thought at what the best version here for SA jobs, and maybe we need several, but why pick the lowest supported version here?

Reply:
Because 7.1 is most likely we will forget about and we need a kick from Psalm to remind us if we accidentally use some >7.1 functionality.

Reply:
Okay, that make sense and since for now Psalm supports 7.1, it does not mean we will not get the latest updates 👍